lib/inputs/file-pattern/static.ts

```diff
@@ -34,7 +34,8 @@ export function generateExtractAction(
   return {
     actionName: "find-files-by-pattern",
     filePathMatches: generatePathMatcher(globsInclude, globsExclude),
-    callback: (dataStream) => streamToString(dataStream, "base64"),
+    callback: (dataStream, streamSize) =>
+      streamToString(dataStream, streamSize, "base64"),
```

Comment:
We don't need `streamSize` in `streamToString()` I think! It's not used currently

Reply:
We do, because it's of type `ExtractCallback` and that now has the size. It's confusing because in here we don't use `streamToString` directly but in other cases we do.

Reply:
Maybe I'm not understanding but if I check the implementation of `streamToString` the streamSize parameter is not used

Reply:
yes but because it used in here for example: 
```
const getFallbackOsReleaseAction: ExtractAction = {
  actionName: "os-release-fallback",
  filePathMatches: (filePath) =>
    filePath === normalizePath(OsReleaseFilePath.LinuxFallback),
  callback: streamToString,
};
```
it had to implement `ExtractCallback ` interface, I mean, I could pass undefined, if that is what you mean? Because it's not used, that is true

Reply:
Aa I see 👍

Reply:
I think it's my bad here, we shouldn't have written `callback: streamToString,` but instead be more explicit like: `callback: async (stream) => streamToString(stream)` (and in any other place that does this).

Reply:
Yes :-) Small refactor opportunity here for sure 👍